src/components/application_manager/src/commands/command_notification_impl.cc

```diff
@@ -63,16 +63,16 @@ bool CommandNotificationImpl::CleanUp() {
 
 void CommandNotificationImpl::Run() {}
 
-void CommandNotificationImpl::SendNotification() {
+void CommandNotificationImpl::SendNotification(const bool final_message) {
   (*message_)[strings::params][strings::protocol_type] = mobile_protocol_type_;
   (*message_)[strings::params][strings::protocol_version] = protocol_version_;
   (*message_)[strings::params][strings::message_type] =
       static_cast<int32_t>(application_manager::MessageType::kNotification);
 
-  LOG4CXX_INFO(logger_, "SendNotification");
+  LOG4CXX_INFO(logger_, "SendNotification: final_message = " << final_message);
```

Comment:
@mvorobio Use std::boolalpha for boolean in LOG

Reply:
@AByzhynar please check https://github.com/LuxoftSDL/sdl_core/pull/18/commits/7bc1f2ce4c04d80778a833d7022995d806ce5788